src/computed/transaction.js

```diff
@@ -12,8 +12,9 @@ const ComputedTransaction = store => {
       const t = transactions ? transactions.slice() : [];
       const p = payments ? payments.slice() : [];
       const i = invoices ? invoices.slice() : [];
-      const all = [].concat(t, p, i);
+      let all = [].concat(t, p, i);
       all.sort((a, b) => b.date.getTime() - a.date.getTime());
+      all = all.slice(0, 100);
```

Comment:
It's fine to do the slice in the return value (line 31). The performance bottleneck isn't V8 but rather the DOM.